src/client.js

```diff
@@ -700,9 +696,11 @@ class RPMOSTreeDBusClient {
                                     }
                                 });
                             transaction_client.call("/", TRANSACTION, "Start");
```

Comment:
Failures from this call won't be caught in the fail unless this is returned.

Reply:
Right, the code doesn't change that. Perhaps we should do that though, good catch!